kolibri/plugins/coach/assets/src/app.js

```diff
@@ -53,9 +54,23 @@ class CoachToolsModule extends KolibriModule {
         },
         {
           name: PageNames.EXAMS,
-          path: '/:class_id/exams',
+          path: '/:classId/exams',
           handler: (toRoute, fromRoute) => {
-            actions.showExamsPage(store, toRoute.params.class_id);
+            examActions.showExamsPage(store, toRoute.params.classId);
+          },
+        },
+        {
+          name: PageNames.CREATE_EXAM,
```

Comment:
for sub pages (like I had been using for `recents`), I think we made an architectural decision to do this through the routing rather than through the routes.

I was getting some console errors saying that there was a missing param when I did this, but it seems to be gone. Do you get anything like this?

[How it's done in actions](https://github.com/learningequality/kolibri/pull/1122/files/7f78076a9846dc2e8747214b4f9be71373c497a8#diff-fea9e3e75d8fe0e1e96da5074cd65528R86)
[How it's done in app.js](https://github.com/indirectlylit/kolibri/blob/7f78076a9846dc2e8747214b4f9be71373c497a8/kolibri/plugins/coach/assets/src/app.js#L37)